lib/store/connectionsSlice.ts

```diff
@@ -0,0 +1,78 @@
+import { Connection } from "..";
+import { StateCreator, StoreApi, UseBoundStore, create } from "zustand";
+
+export interface ConnectionsSlice {
+    connections: {
+        websocket: Connection;
+        boards: Connection[];
+    }
+    setWebSocketConnection: (isConnected: boolean) => void;
+    setBoardConnections: (connections: Array<Connection>) => void;
```

Comment:
```suggestion
    setConnections: (connections: Array<Connection>) => void;
    setConnection: (connection: Connection) => void;
```

see HyperloopUPV-H8/h9-backend#19. This is to make updating connections easier